src/change_stream/options.rs

```diff
@@ -114,6 +114,7 @@ pub enum FullDocumentType {
     /// The
     /// [`ChangeStreamEvent::full_document`](crate::change_stream::event::ChangeStreamEvent::full_document)
     /// field will be populated with a copy of the entire document that was updated.
+    #[serde(rename = "updateLookup")]
```

Comment:
This seems like something that ought to have been caught by testing, are the spec tests missing coverage of this? If so, we should file a DRIVERS ticket.

Reply:
Filed DRIVERS-2201.